src/runtime_src/tools/xclbinutil/XclBinClass.cxx

```diff
@@ -1649,28 +1669,32 @@ XclBin::reportInfo(std::ostream &_ostream, const std::string & _sInputFile, bool
 static void
 parsePSKernelString(const std::string& encodedString,
                     std::string& symbol_name,
-                    unsigned int& num_instances,
+                    unsigned long& num_instances,
                     std::string& path_to_library)
 // Line being parsed:
 //   Syntax: <symbol_name>:<instances>:<path_to_shared_library>
 //   Example: myKernel:3:./data/mylib.so
 //
 // Note: A file name can contain a colen (e.g., C:\test)
 {
-  const std::string delimiters = ":";      // Our delimiter
+  const std::string delimiters = ":";  
 
   // Working variables
   std::string::size_type pos = 0;
   std::string::size_type lastPos = 0;
   std::vector<std::string> tokens;
 
-  // Parse the string until the entire string has been parsed or 3 tokens have been found
+  // Parse the string until the entire string has been parsed or MAX_TOKENS tokens have been found
+  constexpr size_t MAX_TOKENS = 3;
```

Comment:
I am vary of UPPER_CASE variables subject to preprocessor

Reply:
Refactored to use camel case.